MSProject/UIImageView+Wolox.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  UIImageView+Wolox.swift
```

Comment:
Maybe consider creating an `Extensions` folder for all these files. Also, here at Wolox we tend to name these something like `UIImageViewExtension`, but that's just styling for better searchability (is _searchability_ a real word?)

Reply:
Agree